packages/block-editor/src/components/media-placeholder/index.native.js

```diff
@@ -90,13 +90,13 @@ function MediaPlaceholder( props ) {
 	let instructions = labels.instructions;
 	if ( instructions === undefined ) {
 		if ( isImage ) {
-			instructions = __( 'ADD IMAGE' );
+			instructions = __( 'Add image' );
 		} else if ( isVideo ) {
-			instructions = __( 'ADD VIDEO' );
+			instructions = __( 'Add video' );
 		} else if ( isAudio ) {
-			instructions = __( 'ADD AUDIO' );
+			instructions = __( 'Add audio' );
 		} else {
-			instructions = __( 'ADD IMAGE OR VIDEO' );
+			instructions = __( 'Add image or video' );
```

Comment:
Note for reviewers:

Using all-caps is extremely bad for a11y. Many screen-readers don't properly recognize all-caps text as normal text, and instead these words are read-out letter-by-letter, as if they were acronyms.